doc/modules/mod_core.html

```diff
@@ -614,11 +614,11 @@ <h3><a name="DebugLevel">DebugLevel</a></h3>
 <p>
 <hr>
 <h3><a name="DefaultAddress">DefaultAddress</a></h3>
-<strong>Syntax:</strong> DefaultAddress <em>ip-address|dns-name</em> <em>[ip-address|dns-name ...]</em><br>
+<strong>Syntax:</strong> DefaultAddress <em>ip-address|dns-name|device/interface-name [ip-address|dns-name|device/interface-name ...]</em><br>
 <strong>Default:</strong> None<br>
 <strong>Context:</strong> server config<br>
 <strong>Module:</strong> mod_core<br>
-<strong>Compatibility:</strong> 1.2.7rc1 and later
+<strong>Compatibility:</strong> 1.2.7rc1 and later (device/interface name only supported in 1.3.5rc1 and later)
```

Comment:
I admit to a preference of having the description of the directive have a paragraph about which parameters are supported in which releases -- mostly because I feel that folks will actually _read_ those paragraphs; these directive headers are easy for the eye to just skip/skim over when sight-reading.  That's why the current docs use this approach.

Reply:
No worries.  Apologies for the delay in reviewing this further.

Right now, I'm wondering if we can choose either "device" or "interface" (rather than both), and align on that one word.  As a user, which term would be more natural/expected to you, for this documentation?

Reply:
Being a Windows user, I am more familiar with the interface term, but that is only really from the use of the `netsh` console app.  Also I guess it really depends on what you mean.
Another term I am used to is `adapter`, as I usually see that in `Network & Internet Settings` where it has `Change my adapter options`, or in `Network and Sharing Centre` it has `Change adapter settings`.
Network interface, network device, and network adapter could be all interchangeable, but perhaps could also mean different things.  For example device could mean the actual network card, while interface could be the thing the networking messages are sent to, and adapter could effectively be the thing that connects those two.  I also could be completely wrong about this.  I just looked at wikipedia and it appears to say that an adapter is a N.I.C. [https://en.wikipedia.org/wiki/Network_adapter](https://en.wikipedia.org/wiki/Network_adapter)  It also calls it a "Physical network interface".

Perhaps the easiest way is if you had a glossary page, we could choose one of the words and have it be a link to the glossary, perhaps even have it be a pop-up like in wikipedia, although that may be asking a lot, and the glossary could explain this further and even state that often these words are used interchangeably.
Another option would be to instead link to an outside source, e.g. wikipedia, or dictionary.com, for the definition.
A third option would be to have a light glossary with links to an outside source.

If you are asking me what word I think would be better for the average user, I feel like it is adapter, however it does not really feel like it would lend itself to things like `eth0`.  I think interface would be better for that, but I am a more technical person, having been employed as a programmer analyst/developer for over 20 years.  I fear that some people may not know what interface is, which is why I suggest the glossary idea.
I also think that since I have spent limited time in the Linux world, perhaps I am not the best person to ask as I feel that it would be better to use terminology that Linux users are familiar with.